components/ExerciseDisplayed/index.js

```diff
@@ -50,15 +51,19 @@ export default function ExerciseDisplayed({
       {showDetails[index] ? (
         <>
           <StyledParagraphPadding>
-            Difficulty: {dat?.difficulty}
+            <em>Difficulty:</em> {dat?.difficulty}
           </StyledParagraphPadding>
-          <StyledParagraphPadding>Muslce: {dat?.muscle}</StyledParagraphPadding>
-          <StyledParagraphPadding>Type: {dat?.type}</StyledParagraphPadding>
           <StyledParagraphPadding>
-            Equipment: {dat?.equipment}
+            <em>Muslce:</em> {dat?.muscle}
           </StyledParagraphPadding>
           <StyledParagraphPadding>
-            Instructions: {dat?.instructions}
+            <em>Type:</em> {dat?.type}
+          </StyledParagraphPadding>
+          <StyledParagraphPadding>
+            <em>Equipment:</em> {dat?.equipment}
+          </StyledParagraphPadding>
+          <StyledParagraphPadding>
+            <em>Instructions:</em> {dat?.instructions}
```

Comment:
not sure what "dat" relates to, so just a suggestion: name it out :) for me this could be "date" but also "data" - don't want to search through all files to see what it actually means - the instuctions from a fetched api or the manually added data assigned to a date. 

just a suggestion for easier life of reviewers :)

Reply:
good point